src/signalr/hubs/Hub.js

```diff
@@ -0,0 +1,104 @@
+// @flow
```

Comment:
These files basically copy/pasta from brewskey.admin project. I may modify this a bit, when we will come to real events imlementations.